cmd/common_test.go

```diff
@@ -78,7 +78,7 @@ func TestCreateConfig(t *testing.T) {
 	// Keys exists
 	// type KeyPurposeEthMsgAuth
 	idSrv := ctx[identity.BootstrappedDIDService].(identity.ServiceDID)
-	pk, _, err := secp256k1.GetEthAuthKey(cfg.GetEthAuthKeyPair())
+	pk, _, err := secp256k1.GetSigningKeyPair(cfg.GetEthAuthKeyPair())
```

Comment:
Aren't we getting rid of ethauthkey?

Reply:
Yes I believe so, I rather do that as part of a new PR. We need to coordinate with the UI as well.